src/main/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatio.java

```diff
@@ -0,0 +1,89 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ExomeAnalysisProgramGroup;
+import org.broadinstitute.hellbender.utils.plotter.CopyRatioSegmentedPlotter;
+
+@CommandLineProgramProperties(
+        summary = "Create plots of genomic data",
+        oneLineSummary = "Create plots of genomic data",
+        programGroup = ExomeAnalysisProgramGroup.class
+)
+public final class PlotSegmentedCopyRatio extends CommandLineProgram {
+
+    public static final String SAMPLE_NAME_LONG_NAME = "sampleName";
+    public static final String SAMPLE_NAME_SHORT_NAME = "S";
+
+    public static final String TARGETS_FILE_LONG_NAME = "targets";
+    public static final String TARGETS_FILE_SHORT_NAME = "T";
+
+    public static final String PRE_TANGENT_TARGETS_FILE_LONG_NAME = "preTangent";
+    public static final String PRE_TANGENT_TARGETS_FILE_SHORT_NAME = "P";
+
+    public static final String SEGMENT_FILE_LONG_NAME = "segments";
+    public static final String SEGMENT_FILE_SHORT_NAME = "seg";
+
+    public static final String LOG2_LONG_NAME= "log2Input";
+    public static final String LOG2_SHORT_NAME = "log";
+
+    @Argument(
+            doc = "Name of the sample being segmented",
+            shortName = SAMPLE_NAME_SHORT_NAME,
+            fullName = SAMPLE_NAME_LONG_NAME,
+            optional = false
+    )
+    protected String sampleName;
+
+    @Argument(
+            doc = "Genomic targets file after tangent normalization has been applied",
+            shortName = TARGETS_FILE_SHORT_NAME,
+            fullName =  TARGETS_FILE_LONG_NAME,
+            optional = false
+    )
+    protected String tangentFile;
+
+    @Argument(
+            doc = "Genomic targets before tangent normalization file",
+            shortName = PRE_TANGENT_TARGETS_FILE_SHORT_NAME,
+            fullName =  PRE_TANGENT_TARGETS_FILE_LONG_NAME,
+            optional = false
+    )
+    protected String preTangentFile;
+
+    @Argument(
+            doc = "Genomic targets file",
```

Comment:
Did this have to be the same target file used by other processes?  Also, do we need this?  The target (cr ratio) file now includes everything found in the target file...